src/main/java/io/strimzi/kafka/access/internal/KafkaListener.java

```diff
@@ -117,6 +129,9 @@ public Map<String, String> getConnectionSecretData() {
         data.put("securityProtocol", encodedSecurityProtocol);
         //Spring settings
         data.put("bootstrap-servers", bootstrapServers);
+        if (this.tls) {
+            Optional.ofNullable(this.caCertSecret).map(secretData -> secretData.get("ca.crt")).ifPresent(cert -> data.put("ssl.truststore.crt", cert));
```

Comment:
Elsewhere you format these chained method calls over multiple lines.

Reply:
I think it does read easier over several lines so I'll reformat it